app/Http/Resources/StatusResource.php

```diff
@@ -30,23 +30,26 @@ public function toArray($request): array {
             'likes'          => (int) $this->likes->count(),
             'liked'          => (bool) $this->favorited,
             'isLikable'      => Gate::allows('like', $this->resource),
+            'client'         => [
```

Comment:
I would prefer a ClientResource, so that all api consumers will get the name and url of an app